packages/vue-app/template/index.js

```diff
@@ -166,20 +166,23 @@ async function createApp(ssrContext) {
 
   // Plugin execution
   <%= isTest ? '/* eslint-disable camelcase */' : '' %>
-  <% plugins.filter(p => p.mode === 'all').forEach((plugin) => { %>
-  if (typeof <%= plugin.name %> === 'function') await <%= plugin.name %>(app.context, inject)<% }) %>
-
-  <% if (plugins.filter(p => p.mode === 'client').length) { %>
-  if (process.client) {
-    <% plugins.filter(p => p.mode === 'client').forEach((plugin) => { %>
-    if (typeof <%= plugin.name %> === 'function') await <%= plugin.name %>(app.context, inject)<% }) %>
-  }<% } %>
-
-  <% if (plugins.filter(p => p.mode === 'server').length) { %>
-  if (process.server) {
-    <% plugins.filter(p => p.mode === 'server').forEach((plugin) => { %>
-    if (typeof <%= plugin.name %> === 'function') await <%= plugin.name %>(app.context, inject)<% }) %>
-  }<% } %>
+  <% if (plugins.length) { %>
```

Comment:
Is this check required? Calling for each on a zero length array should add no new lines

Reply:
ye, true story :) meaningless optimization

Reply:
changed

Reply:
NP. Thanks :)